libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -3845,6 +3845,35 @@ MAV_RESULT GCS_MAVLINK::handle_command_do_set_home(const mavlink_command_long_t
     return MAV_RESULT_ACCEPTED;
 }
 
+MAV_RESULT GCS_MAVLINK::handle_command_component_arm_disarm(const mavlink_command_long_t &packet)
+{
+    if (is_equal(packet.param1,1.0f)) {
+        if (AP::arming().is_armed()) {
+            // already armed; we're idempotent:
```

Comment:
idempotent ???

Reply:
google !!!   :-P

It basically means you can arm as many times as you like and you end up in the same place.  This is important with mavlink because the GCS might send an "arm" message 2 or 3 times and we don't want to respond "arming failed" when we're actually armed!

Reply:
I'd vote for sticking to english that most people understand.  it probably goes without saying that comments are there to help people understand the code..

Reply:
Problem solved.